Conceptpower+Spring/src/integration-test/resources/output/conceptWithWordPosAndEqualTo.xml

```diff
@@ -0,0 +1 @@
_to><digitalHPS:synonym_ids>[]</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id></digitalHPS:wordnet_id></digitalHPS:conceptEntry><digitalHPS:conceptEntry><digitalHPS:id concept_id="WID-14930670-N-02-1-dodecanol" concept_uri="http://www.digitalhps.org/concepts/WID-14930670-N-02-1-dodecanol">http://www.digitalhps.org/concepts/WID-14930670-N-02-1-dodecanol</digitalHPS:id><digitalHPS:lemma>1-dodecanol</digitalHPS:lemma><digitalHPS:pos>noun</digitalHPS:pos><digitalHPS:description>a colorless insoluble solid alcohol used to make detergents and pharmaceuticals</digitalHPS:description><digitalHPS:conceptList>WordNet</digitalHPS:conceptList><digitalHPS:creator_id></digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to><digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to></digitalHPS:similar_to><digitalHPS:synonym_ids>WID-14930670-N-01-lauryl_alcohol,</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id>WID-14930670-N-02-1-dodecanol</digitalHPS:wordnet_id></digitalHPS:conceptEntry><digitalHPS:conceptEntry><digitalHPS:id concept_id="WID-04502851-N-02-.22" concept_uri="http://www.digitalhps.org/concepts/WID-04502851-N-02-.22">http://www.digitalhps.org/concepts/WID-04502851-N-02-.22</digitalHPS:id><digitalHPS:lemma>.22</digitalHPS:lemma><digitalHPS:pos>noun</digitalHPS:pos><digitalHPS:description>a .22 caliber firearm (pistol or rifle)</digitalHPS:description><digitalHPS:conceptList>WordNet</digitalHPS:conceptList><digitalHPS:creator_id></digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to><digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to></digitalHPS:similar_to><digitalHPS:synonym_ids>WID-04502851-N-01-twenty-two,</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id>WID-04502851-N-02-.22</digitalHPS:wordnet_id></digitalHPS:conceptEntry><digitalHPS:conceptEntry><digitalHPS:id concept_id="WID-00475142-N-02-1-hitter" concept_uri="http://www.digitalhps.org/concepts/WID-00475142-N-02-1-hitter">http://www.digitalhps.org/concepts/WID-00475142-N-02-1-hitter</digitalHPS:id><digitalHPS:lemma>1-hitter</digitalHPS:lemma><digitalHPS:pos>noun</digitalHPS:pos><digitalHPS:description>a game in which a pitcher allows the opposing team only one hit</digitalHPS:description><digitalHPS:conceptList>WordNet</digitalHPS:conceptList><digitalHPS:creator_id></digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to><digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to></digitalHPS:similar_to><digitalHPS:synonym_ids>WID-00475142-N-01-one-hitter,</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id>WID-00475142-N-02-1-hitter</digitalHPS:wordnet_id></digitalHPS:conceptEntry></conceptpowerReply>
```

Comment:
Similar and equal to search return all search results. But this is fixed in develop branch. So based on the search condition, concept entries will be returned.